doc/reference/configuration.rst

```diff
@@ -1088,9 +1088,15 @@ ConsoleExtension
 
 
 
+Verbosity level
+
+
 **Default**: ``32``
 
 
+**Allowed values**: 16, 32, 64, 128, 256
```

Comment:
this might be a bit misleading, currently phpactor's verbosity is rather binary (either it shows logs or it doesn't). could be wrong though.

Reply:
I am sure that phpactor isn't the only project who only uses verbose and non verbose logging. But from the documentation standpoint this is correct. You could add it to the description though, that only <16 and >16 are important values.